test/e2e/testdata/config/config-example-with-extensions.yml

```
@@ -0,0 +1,46 @@
+version: 0.1
+# Modified based on https://github.com/oras-project/distribution/blob/rc2/cmd/registry/config-dev.yml
+log:
+  fields:
+    service: registry
+storage:
+  delete:
+    enabled: true
+  cache:
+    blobdescriptor: inmemory
+  filesystem:
+    rootdirectory: /opt/data/registry-root-dir
+http:
+  addr: :5000
+  headers:
+    X-Content-Type-Options: [nosniff]
+health:
+  storagedriver:
+    enabled: true
+    interval: 10s
+    threshold: 3
+auth:
+  htpasswd:
+    realm: test-basic
+    path: /etc/docker/registry/passwd # hello:oras-test
```

And the bcrpyt password need to be changed too.
```suggestion
    path: /etc/docker/registry/passwd # hello:nonation-test
```